crypto/mem_sec.c

```diff
@@ -437,21 +434,19 @@ static int sh_init(size_t size, size_t minsize)
     pgsize = PAGE_SIZE;
 #endif
     sh.map_size = pgsize + sh.arena_size + pgsize;
-    if (1) {
+
 #ifdef MAP_ANON
+    sh.map_result = mmap(NULL, sh.map_size,
+                         PROT_READ|PROT_WRITE, MAP_ANON|MAP_PRIVATE, -1, 0);
+#else
+    sh.map_result = MAP_FAILED;
+    /* Temporary use |ret| for file descriptor (assumes goto err returns 0) */
```

Comment:
Better to have a local variable just for this.
Somebody will eventually modify this code and not notice this double use.

Reply:
Done